### DISCUSSION TITLE
Connect from Public IP -> VPS

### DISCUSSION THREAD

**Discussion by SupaYoshi** [haugene/docker-transmission-openvpn#2085] [Q&A]
access from public IP possible?

I have Transmission running at a VPS, I want to connect from my local network @home to the transmission client, i tried setting     - LOCAL_NETWORK=mypublicip/32 but this did not work unfortunatly,

when i setup wireguard to the VPS, and I put in the VPN client IP range , it works... how can I set it so that I can connect from my home public IP? I have a firewall setup on the host to block access from other IP's, is this possible?

**Comment by SupaYoshi** [comment]
environment:
    - TRANSMISSION_DOWNLOAD_DIR=/download
    - TRANSMISSION_INCOMPLETE_DIR=/opt/downloads/torrent/incomplete
    - TRANSMISSION_HOME=/config
    - TRANSMISSION_UTP_ENABLED=false
    - TRANSMISSION_PEER_PORT=49545
    - TRANSMISSION_PORT_FORWARDING_ENABLED=true
    - TRANSMISSION_IDLE_SEEDING_LIMIT_ENABLED=true
    - TRANSMISSION_IDLE_SEEDING_LIMIT=60
    - TRANSMISSION_RATIO_LIMIT=2
    - TRANSMISSION_RATIO_LIMIT_ENABLED=true
    - OPENVPN_PROVIDER=slickvpn
    - OPENVPN_CONFIG=config-country
    - OPENVPN_USERNAME=user
    - OPENVPN_PASSWORD=pass
    - OPENVPN_OPTS=--inactive 3600 --ping 10 --ping-exit 60
    - LOCAL_NETWORK=10.67.41.0/24

**Comment by ksurl** [comment]
you need to set the local_network to the local network of the vps. It should either give you a local one like 10.x or just use the public ip of your vps. then if your vps has a firewall, you need to allow it as well. if you are already port mapping 9091, docker bypasses ufw and directly adds it to iptables. another way to do it to use a persistent ssh tunnel to do local forwarding over ssh with autossh or a script that runs `ssh -f -N` and then you don't need a vpn client and connect to localhost:port. local network has nothing to do with your client side. only where the container is running (the vps)
